routes/socket/user-events.js

```diff
@@ -2366,6 +2366,24 @@ module.exports.handleModerationAction = (socket, passport, data, skipCheck, modU
 				case 'disableLimitNewPlayers':
 					limitNewPlayers.status = false;
 					break;
+				case 'removeContributor':
+					if (isSuperMod) {
+						Account.findOne({ username: data.userName })
+							.then(account => {
+								if (account) {
+									account.isContributor = false;
+									account.save(() => {
+										logOutUser(account.username);
+									});
+								} else {
+									socket.emit('sendAlert', `No account found with a matching username: ${data.userName}`);
+								}
+							})
+							.catch(err => {
+								console.log(err);

```

Comment:
I usually have a 2nd argument (String) of what that error was.

Reply:
roger that and thanks

Reply:
um the others before i came to this were only one liners... what exactly are you wanting here and on which actions??